join-us/our-recruitment-process.md

```diff
@@ -166,13 +166,10 @@ We are always on the look-out for great people to join us so don't hesitate to s
 
 We are currently hiring for the following positions:
 
-- [Lead Android Developer](openings/lead-android-developer.md)
-- [Lead iOS Developer](openings/lead-ios-developer.md)
 - [Lead Web Developer](openings/lead-web-developer.md)
 - [Web Developer](openings/web-developer.md)
 - [Ruby/Rails Developer (Bangkok)](openings/ruby-rails-developer-bangkok.md)
 - [Ruby/Rails Developer (Da Nang)](openings/ruby-rails-developer-danang.md)
 - [Technical Product Owner](openings/technical-product-owner.md)
-- [Talent Manager](openings/talent-manager.md)
```

Comment:
The actual markdown files would need to be removed too. Otherwise, users can still find them and apply.

Reply:
Got it, I will remove all the old files and update the positions according to our job site (by location) as well.

Reply:
Hi @olivierobert, all markdown files have been edited/deleted/created according to new vacancies. Please review and let me know if anything else. Thank you.